docs/quick_tour/sqla_demo/sqla_demo/models/meta.py

```diff
@@ -0,0 +1,46 @@
+from sqlalchemy import engine_from_config
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.schema import MetaData
+import zope.sqlalchemy
+
+NAMING_CONVENTION = {
```

Comment:
Maybe a comment here why this name convention has been chosen as it is unclear from the context and novice like me may ponder it. Is it arbitrary choice by the authors, why not SQLAlchemy default or does it base itself on some other existing convention?